3761/1/page_parser.rb

```diff
@@ -0,0 +1,58 @@
+class PageParser
+  attr_reader :page, :parser_data
+
+  PARSER_DATA = { used_by: 0, watch: 0, star: 0, fork: 0, contributors: 0,
+                  issues: 0 }.freeze
+
+  def initialize(page)
+    @page = page
+    @parser_data = {}
+  end
+
+  def self.data_without_used(page)
+    page_parser = new(page)
+    page_parser.find_first_line
+    page_parser.find_second_line
+    page_parser.find_third_line
+    page_parser.parser_data
+  end
+
+  def self.used_by(page)
```

Comment:
If you create an object here
https://github.com/rubizza-camp/summer-2019/pull/311/files#diff-e88a122137a434cc467e7b8f33cf7aadR13
mb it's better to use it? and don't call it on Class with argument